src/QMCWaveFunctions/Jastrow/RPAJastrow.cpp

```diff
@@ -173,7 +193,9 @@ void RPAJastrow::makeShortRange()
   // J2OrbitalSoA<BsplineFunctor<RealType> > *j2 = new J2OrbitalSoA<BsplineFunctor<RealType> >(targetPtcl,IsManager);
   TwoBodyJastrowOrbital<BsplineFunctor<RealType>>* j2 =
       new TwoBodyJastrowOrbital<BsplineFunctor<RealType>>(targetPtcl, IsManager);
-  size_t npts    = 12;
+  size_t nparam  = 12; // number of Bspline parameters
```

Comment:
Hard coded number of bspline knots gives me the willies.  Why 12, and will this come back to bite us if the cell geometry is changed significantly?  Perhaps a comment explaining why 12, or increasing it to a larger, safer number.

Reply:
12 was there before I touched the file. I intend to make it an input parameter eventually. Should I do it in this PR?

Reply:
Not necessary, but definitely leave a note/comment for posterity.